guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -8,16 +8,13 @@ Use this procedure to configure a repository in a directory on a remote server.
 To create a file type repository in a directory on the base system where {ProjectServer} is installed, see xref:Creating_a_Local_Source_for_a_Custom_File_Type_Repository_{context}[].
 
 .Prerequisites
-ifdef::katello[]
-* You have a server running {EL} 7 or 8 registered to your {Project}.
+ifdef::katello,orcharhino[]
+* You have a server running {EL} 8 registered to your {Project}.
 endif::[]
 ifdef::satellite[]
 * You have a server running {EL} 7 or 8 registered to your {Project} or the Red{nbsp}Hat CDN.
 * Your server has an entitlement to the {RHELServer} and {ProjectName} Utils repositories.
 endif::[]
-ifdef::orcharhino[]
-* You have a server running {EL} registered to your {Project}.
-endif::[]
 * You have installed an HTTP server.
 ifndef::orcharhino[]
 For more information about configuring a web server, see {RHELDocsBaseURL}8/html/deploying_different_types_of_servers/setting-apache-http-server_deploying-different-types-of-servers[Setting up the Apache HTTP web server] in {RHEL}{nbsp}8 _Deploying different types of servers_ or {RHELDocsBaseURL}7/html/system_administrators_guide/ch-web_servers#s1-The_Apache_HTTP_Server[The Apache HTTP Server] in the _{RHEL}{nbsp}7 System Administrator's Guide_.
```

Comment:
I'm against it, because there are major differences between docs structures in 7 and 8. If they just switch the version, they have to look it up again. So it's easier if they jsut get the link.

Reply:
Should we limit that for Satellite? In upstream I don't think it makes sense anymore.

If anything, we should move everything to EL 9 as much as possible.

Reply:
Set to satellite-only for now. I think there are many many links to RHEL 7/8/9 docs, maybe we can handle them in a separate PR.